README.md

```diff
@@ -66,6 +67,29 @@ Then to install buildah follow the steps in this example:
   buildah --help
 ```
 
+In Ubuntu/Debian, you can use this command:
+
+```
+  apt-get -qq update
+  apt-get install software-properties-common
+  add-apt-repository ppa:duggan/bats
+  add-apt-repository ppa:alexlarsson/flatpak
+  add-apt-repository ppa:gophers/archive
+  apt-get install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libostree-dev libseccomp-dev libselinux1-dev
+  apt-get install golang-1.8
```

Comment:
Need to add:   
apt-get install go-md2man

Reply:
So this is just to make sure your system is fully up2date.

Reply:
We don't do this in Fedora case.

Reply:
I've played further, we don't need the dist-upgrade or the shutdown.   However we do need:

apt-get install skopeo-containers

adding it and the go-md2man package to line 78 would be just fine.